code/graphics/opengl/gropenglshader.cpp

```diff
@@ -840,6 +862,8 @@ int opengl_compile_shader(shader_type sdr, uint flags)
 		}
 	}
 
+	int new_shader_shader = new_shader.shader;
+	unsigned int new_shader_flags = new_shader.flags;
```

Comment:
Please use `uint32_t` here to make the code more explicit:
```suggestion
	uint32_t new_shader_flags = new_shader.flags;
```

Reply:
Sure, should shader.flags be updated too?

Reply:
Sure, if you want to but it's not strictly necessary since you didn't touch that code.